content/applications/productivity/studio/models_modules_apps.rst

```diff
@@ -0,0 +1,216 @@
+=========================
+Models, modules, and apps
+=========================
+
+Models, also known as *Objects*, determine the logical structure of a database and how data is
+stored, organized, and manipulated. In other words, a model is a table of information that can be
+linked with other tables. A model usually represents a business concept, such as a *sales order*,
+*contact*, or *product*.
+
+Modules and apps contain various elements, such as models, views, data files, web controllers, and
+static web data. The only difference is semantic: larger, standalone modules are typically called
+apps, whereas modules are usually add-ons to said apps.
```

Comment:
Some modules are also apps, but all apps are also a module.

Reply:
I'm not sure I can explain the difference between apps and modules easily then.
If:
- All apps are modules, and
- Some modules are apps.
Why/at which point some modules start being called apps?

Reply:
.. note::
   All apps are modules. However, larger, standalone modules are typically referred to as apps,
   whereas modules usually refers to add-ons to said apps.

Reply:
"the other modules usually serve as add-ons to said apps"?

Note that not all non-app modules are add-ons either :grin: For example, the `payment` and `delivery` modules are neither apps nor add-ons but "engines"/APIs.